core/angel_heart_status.py

```
@@ -234,6 +234,13 @@ def is_event_wake(self, event) -> bool:
             if chat_id and self._is_silenced(chat_id):
                 return False
 
+            # 开启系统级唤醒词时，main.py 会挂载此标记，等价于点名唤醒
```

Ruff (0.16.1)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)